### PR TITLE
Set up a "Hello world" project with rails-react gem

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
## In this milestone I completed the following : 
   - Created a new Rails app called `'hello-rails-react'`.
   - Initialize your project with Git.
   - Set up your project with `webpacker` the react-rails gems
   - Make sure that your project has a `Postgres` database set up.
   - Create a `table` for storing your messages and create 5 different greetings.
   - Create an `API endpoint` that selects a random greeting from your table (you will need a controller with an action and Rails route).
   - Create a `static` view that will be the root of your app.
   - Create the `App` component with react-router and render it in your static view.
   - Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
   - Create a store, an action and a reducer that will connect to you API `endpoint` to get the random greeting, Using `redux`
   - Display the random greeting in your `Greeting` component.

_Thanks for your feedback 💟_

### CODE REVIEW

File: app/javascript/redux/greetings/greetings.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    type: LOAD_GREETING,
+    payload
+});
+
+export const getGreetings = () => async (dispatch) => {
+    await fetch(URL)
+    .then((res) => res.json())
+    .then((greeting) => dispatch(loadGreeting(greeting)))
+    .catch((err) => console.log(err))
+}
```
Comment (rloterh) [inline app/javascript/redux/greetings/greetings.js:20]: - [ ] Greeting is not randomly generated. . There are errors found in the console which are likely traceable to the above error. Kindly improve upon this aspect by ensuring that random greetings are generated when a button is clicked or when the page is refreshed.

**Errors:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hamza, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/redux/greetings/greetings.js (hunk 1 above)
Comment (Hamzaoutdoors) [reply app/javascript/redux/greetings/greetings.js:20]: I checked and re-checked I have no errors in my part and all things working well as described in the requirements as you can see in the screenshot below 

Kazam_screenshot_00007
 - Did you run `rails db:seed` before starting the server ?

**Review by oloomoses — APPROVED** [review]
Hi @Hamzaoutdoors, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/redux/greetings/greetings.js (hunk 1 above)
Comment (rloterh) [reply app/javascript/redux/greetings/greetings.js:20]: @Hamzaoutdoors 
Absolutely.. The seed command was run

**Pull request merged**